demo/calibrationManager.js

```diff
@@ -0,0 +1,97 @@
+export async function storeCalibration(s3, bucket, name, callback) {
+  // TODO
+}
+
+export async function loadRtk2Vehicle(s3, bucket, name, callback) {
+
+  // TODO Hardcoded defaults for now
+  let rtk2Vehicle = {
+    x:0, y:0, z:-2,
+    roll:0, pitch:0, yaw:Math.PI/2
+  }
+
+  callback(rtk2Vehicle);
+}
+
+export async function loadVelo2Rtk(s3, bucket, name, callback) {
+  const tstart = performance.now();
+  //is name here the dataset name? We should be more careful about that....
+  if (s3 && bucket && name) {
+    (async () => {
+      const objectName = `${name}/7_Cals/extrinsics.txt`//`${name}/3_Assessments/gaps.fb`;//fb schema from s3
+
+      s3.getObject({Bucket: bucket,
```

Comment:
Since you're already in an async function, might as well do `await s3.getObject(...).promise()` inside a `try ... catch`.

Reply:
Cool, didn't know about `getObject().promise()` - I updated this